src/index.html

```diff
@@ -39,7 +39,7 @@
 					storeConsentGlobally: false,
 					storePublisherConsentGlobally: false,
 					storePublisherData: true,
-					layout: 'modal', // modal, footer, thin
+          layout: 'footer', // modal, footer, thin
```

Comment:
The commit's message is "Remove unused files". Why the commit contains this change?